test/browser_tests/util/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function shouldShouldnt( value ) {
```

Comment:
It just converts `('should/shouldn't)` to boolean. I've added some documentation https://github.com/cfpb/cfgov-refresh/pull/3005/commits/0cc4b04f49867c50f8e0283ca3ed8b60bfc71664.